### PR TITLE
homework_5

### PR DESCRIPTION
**Pull request by Revy-Lagoon09**

### CODE REVIEW

**Comment by Revy-Lagoon09** [comment]
Emm...can someone explain me why i can't use "for in" in 3rd task? Object is copied? Copied. Copied with all internals, task is done, Ofc I read on the forums that 'for in' bad loop for enumerate over the array, but this is a completely different case in my opinion. I have just copied all shit inside the copyObject and i dont rly need to care that "for in" enumerates the fields of an object, not array indexes or values.  I hope I get some answers... It is advisable to explain as simple as possible (yes i'm debil and don't understand)

File: homeworks/vlad.syniakov_Revy-Lagoon09/homework_5/1.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function Summarizer(value) {
```
Comment (markoffden) [inline homeworks/vlad.syniakov_Revy-Lagoon09/homework_5/1.js]: camelCase function naming please...

File: homeworks/vlad.syniakov_Revy-Lagoon09/homework_5/3.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const Obj = {
```
Comment (markoffden) [inline homeworks/vlad.syniakov_Revy-Lagoon09/homework_5/3.js]: naming again, to not capitalize without a need

File: homeworks/vlad.syniakov_Revy-Lagoon09/homework_5/3.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    b: 'c',
+    d: { e: 'f' },
+};
+function innerCopy(mainObj) {
+    const copyObj = {};
+    for ( const key in mainObj) {
+        copyObj[key] = mainObj[key];
```
Comment (markoffden) [inline homeworks/vlad.syniakov_Revy-Lagoon09/homework_5/3.js]: there is nothing about `for...in`, the problem that simply assigning one object to another objects property is not a copy of object, it is actually a copy of kind of "link" to initial object, that's why it's not what wanted

**Comment by markoffden** [comment]
also, you builds checks are failing, this is caused by linter issues, fix them please

**Pull request merged**